osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -129,7 +130,7 @@ private void load(IAPIProvider api, OsuColour colour)
                                 background = new Box
                                 {
                                     RelativeSizeAxes = Axes.Both,
-                                    Colour = user.OnlineID == api.LocalUser.Value.Id && isOnlineScope ? colour.Green : Color4.Black,
+                                    Colour = isUserFriend ? colour.Pink : (user.OnlineID == api.LocalUser.Value.Id && isOnlineScope ? colour.Green : Color4.Black),
```

Comment:
While this follows `GameplayLeaderboardScore` (with a slightly different pink), the website/beatmap overlay uses yellow and doesn't show the color in friend rankings. Unsure what to follow.

Reply:
It might be a better call to just use yellow, since that's been used in game for awhile to show a player whos on your friend's list

Reply:
Yellow is best for consistency.

Reply:
Changed in https://github.com/ppy/osu/pull/31240/commits/47afab8a32fb312601f8d5b18fb6a9cae6de6e97. Marking as resolved.